.coderabbit.yaml

```
@@ -1,101 +1,28 @@
-    - "dist/**"
-    - "build/**"
-    - "coverage/**"
-    - "*.log"
+# レビューに関する全体設定
+reviews:
+  path_instructions:
+    - path: "**/*.{js,jsx,ts,tsx}" # JavaScript/TypeScriptファイル全体に適用
+      instructions: |
+        あなたは経験豊富なソフトウェアエンジニアです。
+        コードレビューを日本語で、親切かつ建設的に行ってください。
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)